packages/trip-form/src/DateTimeSelector/index.tsx

```diff
@@ -236,21 +218,19 @@ export default function DateTimeSelector({
       className={className}
       role="group"
       style={style}
+      baseColor={baseColor}
     >
-      <S.DateTimeSelector.DepartureRow>
+        <Dropdown alignMenuLeft id="date-time-depart-arrive" text={departureOptions.find(opt => opt.isSelected).text} buttonStyle={{ backgroundColor: S.baseColor() || blue[900], borderRadius: "3px 0px 0px 3px", color: "white", height: "45px", border: "0px", padding:"5px 7px" }}>
```

Comment:
Can we make the default gray instead of blue?

Reply:
Also some of these values are pretty fixed and are causing layout problems in some implementations. Any way we can make this more reactive to its environment?